src/isNotValidNumber.js

```diff
@@ -0,0 +1,24 @@
+import { complement } from 'ramda';
+import isValidNumber from './isValidNumber';
+
+/* eslint-disable max-len */
+/**
+ * Checks if value is not a valid `Number` object or is `NaN`,
```

Comment:
If we gonna drop the `Number` prexif, then let's use proper global object naming
```javascript
POSITIVE_INFINITY => Infinity
NEGATIVE_INFINITY => -Infinity